### PR TITLE
When a user says "hello" with no context, the link https://nohello.com/ will be sent.

### PR DESCRIPTION
**Pull request by TheOnlyTails**
Intellij did some auto-refactoring and added some stuff, idk.

### CODE REVIEW

**Comment by Sparky200** [comment]
There should probably be a channel check since people like to say hello in general channels 😉

**Comment by TheOnlyTails** [comment]
True - I'll change it.

**Comment by TheOnlyTails** [comment]
I added a check so it doesn't fire for channels in the general category (#general, #programming, #gaming, #media, and #showcase).

File: src/main/java/com/technovision/technobot/listeners/ExtrasEventListener.java [hunk 1]
```diff
@@ -1,10 +1,15 @@
 package com.technovision.technobot.listeners;
 
+import net.dv8tion.jda.api.entities.GuildChannel;
 import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.dv8tion.jda.internal.entities.CategoryImpl;
+import net.dv8tion.jda.internal.entities.GuildImpl;
+import net.dv8tion.jda.internal.entities.TextChannelImpl;
```
Comment (xf8b) [inline src/main/java/com/technovision/technobot/listeners/ExtrasEventListener.java:8]: why are you using internal classes :concern:

File: src/main/java/com/technovision/technobot/listeners/ExtrasEventListener.java [hunk 2]
```diff
@@ -78,6 +83,17 @@ public void onGuildMessageReceived(@Nonnull GuildMessageReceivedEvent event) {
             event.getChannel().sendMessage(reply).queue();
             event.getMessage().addReaction("\uD83D\uDE20").queue();
             triggered = true;
+        } else if (msg.equalsIgnoreCase("hello")) {
+            List<GuildChannel> categoryGeneral = new CategoryImpl(599345363102597132L, (GuildImpl) event.getGuild())
+                    .getChannels();
+            TextChannelImpl programmingChannel = new TextChannelImpl(739159751518322818L, (GuildImpl) event.getGuild());
```
Comment (xf8b) [inline src/main/java/com/technovision/technobot/listeners/ExtrasEventListener.java:89]: i dont remember if these methods exist but i think you can just use the jda client and get the guilds stuffs

**Review by xf8b — COMMENTED** [review]
internal class :concern:

**Review by xf8b — APPROVED** [review]

**Comment by KingRealzYT** [comment]
> I added a check so it doesn't fire for channels in the general category (#general, #programming, #gaming, #media, and #showcase).

make it so it fires in #programming

**Pull request closed without merging**